src/components/Articles/Articles.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../../api";
+
+const Articles = () => {
+  const [total, setTotal] = useState(0);
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getArticles()
+      .then(({ articles, total_count }) => {
+        setArticles(articles);
+        setTotal(total_count);
+        setIsLoading(false);
+        setIsError(false);
+      })
+      .catch((err) => {
+        setIsLoading(false);
+        setIsError(true);
+      });
+  }, []);
+
+  if (isLoading) return <h1>Loading...</h1>;
+  if (isError) return <h1>Error. This has not worked </h1>;
+
+  return (
+    <>
+      <ul className="articles-container">
+        {articles.map((article) => {
+          return (
```

Comment:
Within this return statement, can we have some sort of link that takes us to each individual article page please?